tests/acceptance/features/apiSharingNg/shareInvitations.feature

```diff
@@ -1537,3 +1537,79 @@ Feature: Send a sharing invitations
       | Manager          | file          | /textfile1.txt |
       | Co Owner         | folder        | FolderToShare  |
       | Manager          | folder        | FolderToShare  |
+
+
+  Scenario Outline: send share invitation for project space to user with different roles
```

Comment:
will you add sharing of resource inside of project space in this PR or other?

Reply:
No, it will be covered in other PRs, this PR is only for space share.